### PR TITLE
Basic Overview Data

### PR DESCRIPTION
**Pull request by AedonGrunGott**
I touched App Holder to do this, so we should merge into main to avoid future conflicts.

### CODE REVIEW

File: client/source/utilities/axiosRequests.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const get = function getByEndpoint(endpoint) {
  * */
 const post = function postBygetByEndpoint(endpoint, json) {
```
Comment (BrettEastman) [inline client/source/utilities/axiosRequests.js]: Very small issue that probably doesn't matter, but should it be postByGetByEndpoint to include 'g' in camel-casing?

**Review by BrettEastman — APPROVED** [review]
Very small thing about camel-casing but otherwise looks good!

File: client/source/utilities/axiosRequests.js (hunk 1 above)
Comment (AedonGrunGott) [reply client/source/utilities/axiosRequests.js]: It shouldn't even be like that, VSCode replace didn't repalce as smoothly as I wanted it to!

File: client/source/components/overview/Overview.jsx [hunk 2]
```diff
@@ -1,34 +1,65 @@
...
+import PhotoBlock from './PhotoBlock.jsx';
+import ProductInfo from './ProductInfo.jsx';
+import ProductText from './ProductText.jsx';
 
-import * as requests from '../../utilities/axiosRequests';
+import ProdContext from '../../ProdContext.js';
 
+import * as requests from '../../utilities/axiosRequests.js';
+
+// EXAMPLE GET PRODUCT BY ID
```
Comment (AedonGrunGott) [inline client/source/components/overview/Overview.jsx:14]: Someone should have called me out on this!

**Pull request merged**